src/ostorlab/agent/message/proto/v3/use/device/device.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto2";
+
+package ostorlab.agent.message.proto.v3.use.device;
+
+
+message LoginPassword {
+  optional string username = 1;
+  optional string password = 2;
+}
+
+message SSHKey {
+  optional string private_key = 1;
+  optional string public_key = 2;
+}
+
+message Credentials {
+  oneof credentials {
+    LoginPassword login_password = 1;
+    SSHKey ssh_key = 2;
+  }
+}
+
+message DeviceRelay {
+  optional string name = 1;
+  optional string address = 2;
+  optional sint32 port = 3;
+}
+
+message Duration {
+  optional int32 nanos = 1;
+  optional int64 seconds = 2;
+}
+
+message Message {
+  optional string device_id = 1;
+  optional string device_type = 2;
```

Comment:
What is device_type?

Reply:
Android Iphone Ipad

Reply:
Is it the same as the platform: `android | ios`
and can it be an enum? instead of a string.